File: Python/Condicionais/maiorMenor.py
nome = input("Qual o seu nome? ")
idade = int(input(f"Seja bem-vindo {nome}, qual a sua idade? "))

if (idade >= 18):
    print(f"Parabéns {nome}! Você já pode tirar carteira de motorista.\n")
else:
    print(f"Você ainda é menor de idade {nome}, mas futuramente poderá tirar sua carteira de motorista.")
    
print(f"\nFim do programa. Volte sempre {nome}!")
print("\n\n")

File: CPP/treino de condicionais/maiorMenor.cpp
//Código que vai verificar se uma pessoa é menor ou maior de idade.
#include <iostream>
#include <string>
using namespace std;

int main(){
    int idade;
    string nome;
    cout << "Ola seja bem-vindo ao meu programa.\n";
    cout << "Vamos verificar se voce e maior ou menor de idade\n";
    cout << "Qual o seu nome? ";
    getline(cin, nome);
    cout << "Digite a sua idade: ";
    cin >> idade;

    cout << "\n\n";

    if (idade >= 18){
        cout << "Certo " << nome << ", Voce e maior de idade.\n";
    } else {
        cout << "Certo " << nome << ", Voce e menor de idade.\n";
    }

    return 0;
}